library/src/webapp/juice/juice-weblearn.js

```diff
@@ -21,8 +21,8 @@ function runExtensions(){
 			// ****************
 			
 			var availServer = "/library-availability/library"; // DAIA server for print availability
-			var availabilityDiv = '<div id="availability"></div>';
-			var insert_avail = new JuiceInsert(availabilityDiv,"span.Z3988","after");
```

Comment:
As long we still have a <span class="Z3988"> in the page this is fine. That special span allows client side reference managers to automatically pull out the references I think.